docker/Dockerfile.ci

```diff
@@ -0,0 +1,22 @@
+
+# Create a base image with the latest packages
+# We must use "debian:bullseye" (and not "debian:buster") as the base runtime image to match the libc version used while building the binaries
+FROM debian:bullseye as updated-base
+
+RUN apt-get update
```

Comment:
this should only need to be run on the builder, since it does not actually upgrade any packages